.prettierrc

```diff
@@ -1,4 +1,8 @@
 {
-  "printWidth": 120,
-  "tabWidth": 4
+    "printWidth": 120,
+    "tabWidth": 4,
+    "trailingComma": "es5",
```

Comment:
Can we leave this off? Can't stand this feature 😅

Reply:
Or at least reformat the files so there are no trailing commas

Reply:
Sure! Sorry I added it, because sometimes there were trailing commas and sometimes not 🤷‍♂️ Maybe this was a mistake by me and my formatter. I'll update it as soon as I can.

Reply:
No worries! Sorry if it seems like I'm nitpicking here 😅. Keep up the great work mate!

Reply:
No problem! I can fully understand that (: